### DISCUSSION TITLE
{{ author: name }} not working

### DISCUSSION THREAD

**Discussion by goodBranchUC** [statamic/cms#6325] [Q&A]
New here and to programming (somewhat)

going through the statamic 3 starting guide.

I'm most likely missing something right in front of my eyes but I just can't see it.
![Screenshot_7](https://user-images.githubusercontent.com/99146550/178379306-91382c47-23dc-48c6-aeb9-ed847e2184c8.png)

![Screenshot_8](https://user-images.githubusercontent.com/99146550/178379402-7ad936a4-86a7-4703-a4b3-73c47a5da90e.png)

![Screenshot_9](https://user-images.githubusercontent.com/99146550/178379504-73cf1758-73c9-45c4-8dec-7ff8afe19701.png)

1.st directions
2nd my copied code
3rd my output

the author doesnt show and the css is not right. i found an older thread on this discontinued though.

**Comment by goodBranchUC** [comment]
Oh I forgot to add this
![Screenshot_10](https://user-images.githubusercontent.com/99146550/178379931-bf0e6804-92c3-4093-bf7d-7406d10e1228.png)

**Reply by JohnathonKoster** [reply]
For a single author, make sure to set Max Items to `1`, otherwise it will return an array/query builder instance 👍

**Reply by goodBranchUC** [reply]
didn't know that. makes perfect sense when this worked

...by {{ author }} {{ name }} {{ /author }}

**Reply by goodBranchUC** [reply]
changed to 1 but {{ author:name }} still doesnt work.

**Reply by JohnathonKoster** [reply]
Does the template still have `{{ author: name }}` (with the space)? If so, does it work if you change it to `{{ author:name }}` without the space? 🤔

**Reply by goodBranchUC** [reply]
I got it. I didn't know that the guide's code snippets were copy/paste-able. so i was using a screen-snippet-to-text software which had been replacing l's with 1's. and a few other very hard to identify until i found out about vscode compare with. thanks for the help.

**Comment by goodBranchUC** [comment]
{{ author: name }} and {{ author:name }} both work now.
